File: MODUL 3/Soal 1/PRAK301_2410817210026-Fadhil_Syahdama_Mahatma_Putra.c
#include <stdio.h>

int main() {
    int haha, hihi, hoho;

    scanf("%d %d %d", &haha, &hihi, &hoho);
    if (haha <= hihi && haha <= hoho) {
        if (hihi <= hoho) {
            printf("%d %d %d\n", haha, hihi, hoho);
        } else {
            printf("%d %d %d\n", haha, hoho, hihi);
        }
    } else if (hihi <= haha && hihi <= hoho) {
        if (haha <= hoho) {
            printf("%d %d %d\n", hihi, haha, hoho);
        } else {
            printf("%d %d %d\n", hihi, hoho, haha);
        }
    } else if (hoho <= haha || hoho<=hihi){
        if (haha <= hihi) {
            printf("%d %d %d\n", hoho, haha, hihi);
        } else {
            printf("%d %d %d\n", hoho, hihi, haha);
        }
    }
}

File: MODUL 3/Soal 1/PRAK301_2410817210026-Fadhil_Syahdama_Mahatma_Putra.py
def main():
    haha, hihi, hoho = input().split()  
    haha, hihi, hoho = int(haha), int(hihi), int(hoho)

    if haha <= hihi and haha <= hoho:
        if hihi <= hoho:
            print(haha, hihi, hoho)
        else:
            print(haha, hoho, hihi)
    elif hihi <= haha and hihi <= hoho:
        if haha <= hoho:
            print(hihi, haha, hoho)
        else:
            print(hihi, hoho, haha)
    else:
        if haha <= hihi:
            print(hoho, haha, hihi)
        else:
            print(hoho, hihi, haha)

main()